pprint/src/main/scala/ammonite/pprint/PPrint.scala

```diff
@@ -126,6 +154,15 @@ object PPrinter {
   implicit def SeqRepr[T: PPrint] = Internals.collectionRepr[T, Seq[T]]
   implicit def SetRepr[T: PPrint] = Internals.collectionRepr[T, Set[T]]
   implicit def MapRepr[T: PPrint, V: PPrint] = Internals.makeMapRepr[Map, T, V]
+  
+  implicit def fullPPrinter[A: PPrint]: PPrinter[Full[A]] = {
+    new PPrinter[Full[A]]{ 
+      def render(wrapper: Full[A], c: Config) = {
+        val pprint = implicitly[PPrint[A]]
+        Iterator("asd") ++  pprint.copy(cfg = c.full).render(wrapper.a)
```

Comment:
Trace for seeing whether this code really executes. Shouldn't have gotten into version control though.